src/chains/arbitrum/vm/opcodes/block/coinbase.ts

```diff
@@ -1,16 +1,17 @@
 import { Opcode } from '@/types';
 
-export const coinbase: Omit<Opcode, 'playgroundLink'> = {
+export const coinbase: Pick<
```

Comment:
As I'm reading these, do you mind using `Omit` instead of `Pick`? `Omit` makes it more readable because you can also see the missing fields (since the chosen fields are already present in the object data below)

Reply:
I like that, makes sense.